src/app/page.tsx

```diff
@@ -1,17 +1,13 @@
 'use client'
+import { HomepageCarousel } from '@/components/Carousel/index'
 import { Container } from 'react-bootstrap'
 
 export default function Home() {
   return (
-    <Container>
-      <h1>Hello, World!</h1>
-      <p>
-        Lorem ipsum, dolor sit amet consectetur adipisicing elit. Inventore
-        minima ea atque quas sunt laudantium, nihil suscipit reprehenderit non
-        doloremque dolorem totam hic sapiente rem culpa aut autem quibusdam.
-        Atque!
-      </p>
-      <div style={{ height: '2000px' }}></div>
-    </Container>
+    <main>
+      <Container>
+        <HomepageCarousel />
+      </Container>
+    </main>
```

Comment:
`<main>` is already defined in `layout` we can remove from here:
```suggestion
      <Container>
        <HomepageCarousel />
      </Container>
```

Reply:
Ok, done.